server/public/model/user.go

```diff
@@ -142,7 +142,26 @@ func (u *User) Auditable() map[string]interface{} {
 }
 
 func (u *User) LogClone() any {
-	return u.Auditable()
+	return map[string]interface{}{
+		"id":              u.Id,
+		"create_at":       MillisToString(u.CreateAt),
```

Comment:
This is a minor breaking change and makes the logging of this struct different than others in terms of timestamp format.  This will break any scripts or Prometheus queries.   I think this should be done on a major version and on all models at once, after consensus from R&D group.

Reply:
Do we consider the logged messages part of an API that needs to be protected from breaking changes? Would removing fields also be considered a breaking change?

I'm leaning toward putting this change in the bugfix category as logs are designed to be consumed by humans, and UNIX millis are not human friendly.

> and makes the logging of this struct different than others in terms of timestamp format. This will break any scripts or 

I would love to also update the other structs :wink: There are only 6 of them.

Reply:
One point to add: The `timestamp` field of the log entries also uses a formatted string, not unix millis.

Reply:
I suggest checking with cloud team and/or SRE to see if this change would break anything in their world.